inference-engine/src/plugin_api/cpp_interfaces/interface/ie_internal_plugin_config.hpp

```diff
@@ -45,6 +45,15 @@ DECLARE_CONFIG_KEY(CPU_THREADS_PER_STREAM);
  */
 DECLARE_CONFIG_KEY(FORCE_DISABLE_CACHE);
 
+/**
+ * @brief The name for setting work mode internal in MULTI device plugin option.
+ *
+ * This option should be used with value only:
+ * PluginConfigInternalParams::MULTI_MODE_AUTO or PluginConfigInternalParams::MULTI_MODE_LEGACY
+ */
+DECLARE_CONFIG_KEY(WORK_MODE);
+DECLARE_CONFIG_VALUE(MULTI_MODE_AUTO);
```

Comment:
this value seemigly not used anyway, maybe remove.
also WORK_MODE, better to change to the AUTO_MODE.

pls do that in the next PR (https://github.com/openvinotoolkit/openvino/pull/7076 I guess)

Reply:
thanks, will rename